visivo/commands/deploy.py

```diff
@@ -77,11 +68,11 @@ def deploy(working_dir, user_dir, output_dir, stage, host):
             body = {
                 "name": trace.name,
                 "project_id": project_id,
-                "data_file_id": response.model_dump_json()["id"],
```

Comment:
Ah so this is what was breaking deploy some pydantic 1.0 syntax

Reply:
I think it was a bad replace.

Reply:
I agree, model dump json was a lot more clear than just json